### ISSUE TITLE
Pylance not provding all it's features

### ISSUE THREAD

**Issue by darkscarletspeedster** [microsoft/pylance-release#74]
1. Intellisense not working for numpy and seen from other issues stated it's not working for externally installed libraries
2. The features like function description and auto-importing could not be seen in action.
3. The intellisence that I get do not show the information in the way it states it would, I guess it works according to Visual Studio IntelliCode

**Comment by ajay2605** [comment]
auto import not working for me as well.

**Comment by jakebailey** [comment]
Could you please file distinct issues following our bug template for each? This isn't enough information about each of these to properly investigate.

Numpy is mostly compiled, so we are missing information there until they publish a version with their stubs included.

There are a few open issues about auto-imports, do any of those cover your issue?

I'm not sure what "it states it would" means in the context of intellisense.

**Comment by darkscarletspeedster** [comment]
So when I installed numpy, intellisence wasn't working for it, auto-imports were not working in the sense if that's what the feature means is when you write math in the code it interprets it and adds the required import statement automatically.
While about other issues I did not go through them completely so can't say, but more importantly the intellisence one is more bothering. Also a detailed intellisense view is not shown for in built functions. Have attached an image for reference
![image](https://user-images.githubusercontent.com/32820679/87069874-92ca2d00-c235-11ea-91f4-bf0af39ec90c.png)
